### ISSUE TITLE
Why no just follow your trending lang via your day-to-day rss feed?

### ISSUE THREAD

**Issue by netanelravid** [vitalets/github-trending-repos#113]

**Comment by vitalets** [comment]
Hi @netanelravid ! Could you show example rss link to some lang?

**Comment by netanelravid** [comment]
![image](https://user-images.githubusercontent.com/9547714/36639266-9f65cbba-1a11-11e8-961c-0e614236c302.png)
Personally, I'm using Feedly which is great to follow your desired sources.
The image above is one of several of my feeds - Github trending JS

**Comment by vitalets** [comment]
Thanks for sharing it! I agree RSS is also an option for tracking GitHub trends. I've tried it for some time before creating this project. I think all [alternatives](https://github.com/vitalets/github-trending-repos#alternatives) mentioned in readme have own pros and cons, and allows people to choose the most comfortable way.

Personally I found getting GitHub trends via notifications more convenient:
* I'm dealing with notifications anyway and trends seamlessly integrated in my daily workflow
* I'm familiar with GitHub UI
* I don't want to visit another website, setup any tool, register somewhere and etc 
* I can overview trends without loosing the working context
